public-api/src/main/java/org/pmiops/workbench/utils/mappers/ConceptMapper.java

```diff
@@ -0,0 +1,25 @@
+package org.pmiops.workbench.cdr;
+
+import org.mapstruct.Mapper;
+import org.pmiops.workbench.model.MeasurementConceptInfo;
+import org.pmiops.workbench.model.Concept;
+import org.pmiops.workbench.cdr.model.DbMeasurementConceptInfo;
+import org.pmiops.workbench.cdr.model.DbConcept;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+@Mapper(
+        config = MapStructConfig.class,
+        uses = {CommonMappers.class})
+public interface ConceptMapper {
+    MeasurementConceptInfo dbModelToClient(DbMeasurementConceptInfo db);
```

Comment:
Doesn't seem like a db model to client conversion for `MeasurementConceptInfo` belongs in the `ConceptMapper`. Should remove this.

Reply:
This is not needed anymore. Thanks.